gdpr/impl_test.go

```diff
@@ -177,6 +177,73 @@ func TestMalformedConsent(t *testing.T) {
 }
 
 func TestAllowPersonalInfo(t *testing.T) {
+	tests := []struct {
+		description string
+		bidderName  openrtb_ext.BidderName
+		publisherID string
+		gdpr        GDPRState
+		consent     string
+		allowPI     bool
+	}{
+		{
+			description: "Allow PI - Non standard publisher",
+			bidderName:  openrtb_ext.BidderAppnexus,
+			publisherID: "appNexusAppID",
+			gdpr:        YesGDPR,
+			consent:     "BOS2bx5OS2bx5ABABBAAABoAAAABBwAA",
+			allowPI:     true,
+		},
+		{
+			description: "Don't allow PI - No GDPR",
```

Comment:
If GDPR does not apply, why would we not allow PI?

Reply:
Not sure where my head was at here. This mistake is related to the one you pointed out above.